### DISCUSSION TITLE
help with Koreader and menu

### DISCUSSION THREAD

**Discussion by PocahontasBx** [koreader/koreader#10103] [Q&A]
My old Kobo mini doesn't work. I re-initialised the device. Uploading the update with Kobo Desktop doesn't work. So I can't use it. Trying to help and contacting the Kobo company wasn't helpful for me.
I found Koreader and successfully installed it. At the end of the installation process, the message "successful installation" was displayed.  But there is still no menu where I can select the Koreader. Since Nickel isnn't working, I'm blocked. I tried to install the Kobo start menu, but I can't because Nickel doesn't work. 
What can I do?

**Comment by Frenzie** [comment]
You'll probably have better luck over on https://www.mobileread.com/forums/forumdisplay.php?f=247

If your SD card is corrupted, perhaps you can replace it. There's also a thread where people share the required image: https://www.mobileread.com/forums/showthread.php?t=241799&highlight=kobo+mini+card

**Reply by PocahontasBx** [reply]
Thanks a lot @Frenzie 
Iwill try with the required image of Nickel.
But it cannot work without it directly with koreader alone ?

**Reply by Frenzie** [reply]
In theory maybe, but not if there's a hardware problem.
